server/db/schema.js

```diff
@@ -72,6 +76,41 @@ const schema = {
         created_at timestamp with time zone DEFAULT now()
       )`,
     },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.POST_SECONDARY_INSTITUTIONS} (
+        id serial primary key,
+        institute_name varchar(200) not null,
+        health_authority varchar(50) not null,
+        available_seats integer not null,
+        postal_code varchar(10) not null
+      )`,
+    },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.COHORTS} (
+        id serial primary key,
+        cohort_name varchar(200) not null,
+        start_date date not null,
+        end_date date not null,
+        cohort_size integer not null,
+        psi_id integer references ${collections.POST_SECONDARY_INSTITUTIONS},
+        participant_id integer not null,
+        created_at timestamp with time zone DEFAULT now()
+      )`,
+    },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.COHORT_PARTICIPANTS} (
+        id serial primary key,
+        cohort_id integer references ${collections.COHORTS},
+        participant_id integer references ${collections.PARTICIPANTS}
+      )`,
+    },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.PSI_COHORTS} (
```

Comment:
IS PSI -> Cohort a many to many? I thought we discussed this as a 1 to many

Reply:
It's one to many, is there a better way of implementing it?

Reply:
Cohorts should have a FK reference to PSI to indicate the one to many, that way `cohorts` can only have one value (psi_id) to link it to a single PSI

With a linking table we could add many records which would create a many-to-many

Reply:
Okay, that makes sense. How do we get cohorts from a given PSI record in that case?

Reply:
I suppose we can join on Cohorts with a given psi_id, would the performance take a hit at all?

Reply:
Just a simple query: `select * from cohorts where psi_id = 'foo'`

Reply:
I'm agree with @arranfw , we required join table incase of many to many, here participant -> cohort is many to many relationship.